README.md

```diff
@@ -52,29 +53,29 @@ challenges raised by the [CAP Theorem][]. Each application
 can select the right combination of libraries to solve its particular
 distributed problem.
 
-Irmin consists of a core of well-defined low-level data structures
-that specify how data should be persisted and be shared across
-nodes. It defines algorithms for efficient synchronization of those
+Irmin is built on a core of well-defined, low-level data structures that 
+dictate how data should be persisted and shared across nodes. 
+It defines algorithms for efficient synchronisation of those
 distributed low-level constructs. It also builds a collection of
 higher-level data structures that developers can use without knowing
 precisely how Irmin works underneath. Some of these components even
-have a [formal semantics][], including [Conflict-free Replicated
+have [formal semantics][], including [Conflict-Free Replicated
 Data-Types (CRDT)][]. Since it's a part of MirageOS, Irmin does not
-make strong assumptions about the OS environment that it runs in. This
-makes the system very portable: it works well for in-memory databases
-and slower persistent serialization such as SSDs, hard drives, web
+make strong assumptions about the OS environment, which makes the system
+very portable. It works well for in-memory databases
+and slower persistent serialisation, such as SSDs, hard drives, web
 browser local storage, or even the Git file format.
 
 Irmin is primarily developed and maintained by [Tarides][], with
-contributions from many [contributors][] from various
-organizations. External maintainers and contributors are welcome.
+involvement by [contributors][] from various
+organisations. External maintainers and contributors are welcome.
 
 [MirageOS]: https://mirage.io
 [CAP Theorem]: http://en.wikipedia.org/wiki/CAP_theorem
-[formal semantics]: https://kcsrk.info/papers/banyan_aplas20.pdf
-[Conflict-free Replicated Data-Types (CRDT)]: https://arxiv.org/abs/2203.14518
+[Formal Semantics]: https://kcsrk.info/papers/banyan_aplas20.pdf
+[Conflict-Free Replicated Data-Types (CRDT)]: https://arxiv.org/abs/2203.14518
 [Tarides]: https://tarides.com
-[contributors]: https://github.com/mirage/irmin/graphs/contributors
+[Contributors]: https://github.com/mirage/irmin/graphs/contributors
```

Comment:
```suggestion
[contributors]: https://github.com/mirage/irmin/graphs/contributors
```

These markdown links are not actually displayed, but are used in the above paragraphs to avoid having the url in the paragraphs as it's harder to read then... but then they appear in the paragraphs without the capitalization :)